### DISCUSSION TITLE
Функция while или что-то другое?

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#300] [Неопределенное]
Подскажите пожалуйста, почему у админа такая штука открывает правильный linktopanel, а у пользователя открывается не тоже самое:

```
= : while(condition: #fl_contact != ""; action: $insert; action: $linkto)

	insert: insert(table: $#ntn; field: 'contact' = #fl_contact; inserts: "shownewid"; log: true)
	linkto: linkToPanel(table: $#ntn; id: $#shownewid; refresh: true)
```
![Screenshot_4](https://user-images.githubusercontent.com/113771444/210640027-42c42d03-e944-4211-86f1-e290bb92cdfa.jpg)

![Screenshot_3](https://user-images.githubusercontent.com/113771444/210639894-12bebe2c-a068-45fd-b4e1-2cece8b30d8d.jpg)

**Comment by ttm-alexeypolunin** [comment]
Что в поле Контакт?

**Reply by ttm-alexeypolunin** [reply]
И Реквизиты это Временная таблица?

**Reply by mihail-cornev** [reply]
Поле контакт это Селект по таблице Контакты.
А реквизиты это простая таблица с префильтром.

**Reply by mihail-cornev** [reply]
Этот код работает по нажатию кнопки добавить. В простой таблице Реквизиты, есть кнопка Добавить, и по ее нажатию, должна создаваться строка и открываться панель для ее редактирования.

**Reply by ttm-alexeypolunin** [reply]
> Поле контакт это Селек

Какой код-селекта и форматирования у Контакта?

**Reply by mihail-cornev** [reply]
1. Код селекта заполнен.
2. Форматирование просто включено, но не заполнено.
**Сегодня вчерашний глюк исчез, работает вроде как надо.**

4. Видео про глюк кнопки добавления.


![Screenshot_2](https://user-images.githubusercontent.com/113771444/210789022-951e15b3-cd86-43d7-824f-f175608637a1.jpg)
![Screenshot_3](https://user-images.githubusercontent.com/113771444/210789011-fb83b876-b0d6-4256-b05c-49bf6c3ac493.jpg)
[Реквизиты — ERGO.webm](https://user-images.githubusercontent.com/113771444/210789015-8bcfa5a4-20bd-4468-ae4f-6b03c7ef4b12.webm)

**Reply by ttm-alexeypolunin** [reply]
глюк знаем, мне кажется он уже исправлен в последней версии на GH

**Reply by mihail-cornev** [reply]
> глюк знаем, мне кажется он уже исправлен в последней версии на GH

спасибо, обновился.

**Reply by ttm-alexeypolunin** [reply]
Я прочекаю по вашему описанию тогда завтра тк мы меняли кишки панелям и там что-то в индивидуальных селекта например могло криво работать.
